lib/signs_ui_web/views/layout_view.ex

```diff
@@ -1,3 +1,8 @@
 defmodule SignsUiWeb.LayoutView do
   use SignsUiWeb, :view
+  @spec record_sentry?() :: boolean()
+  def record_sentry?() do
+    sentry_env = Application.get_env(:sentry, :environment_name)
+    sentry_env == "dev" || sentry_env == "prod"
```

Comment:
Is there a reason to check for the environment name instead of checking for whether the DSN is configured?

Reply:
Probably not.